test/auction-test.js

```diff
@@ -372,6 +374,94 @@ describe('Auction', function() {
       assert.bufferEqual(root, chain.db.txn.rootHash());
     });
 
+    it('should not have transfer stats in JSON yet', async () => {
+      const ns = await chain.db.getNameStateByName(NAME1);
+      const {stats} = ns.getJSON(chain.height, network);
+      assert.ok(stats.renewalPeriodStart);
+      assert.ok(stats.renewalPeriodEnd);
+      assert.ok(stats.blocksUntilExpire);
+      assert.ok(stats.daysUntilExpire);
+      assert.ok(!stats.transferLockupStart);
+      assert.ok(!stats.transferLockupEnd);
+      assert.ok(!stats.blocksUntilValidFinalize);
+      assert.ok(!stats.hoursUntilValidFinalize);
+    });
+
+    it('should transfer a name', async () => {
+      const mtx = await winner.createTransfer(NAME1, runnerup.getReceive());
+
+      const job = await cpu.createJob();
+      job.addTX(mtx.toTX(), mtx.view);
+      job.refresh();
+      const block = await job.mineAsync();
+      const entry = await chain.add(block);
+      assert(entry);
+      transferBlock = entry.height;
+    });
+
+    it('should be in a transfer state', async () => {
+      const ns = await chain.db.getNameStateByName(NAME1);
+      assert.strictEqual(ns.transfer, transferBlock);
+      assert(ns.transfer !== 0);
+    });
+
+    it('should have transfer stats', async () => {
+      const ns = await chain.db.getNameStateByName(NAME1);
+      const {stats} = ns.getJSON(chain.height, network);
+      assert.ok(stats.renewalPeriodStart);
+      assert.ok(stats.renewalPeriodEnd);
+      assert.ok(stats.blocksUntilExpire);
+      assert.ok(stats.daysUntilExpire);
+      assert.ok(stats.transferLockupStart);
+      assert.ok(stats.transferLockupEnd);
+      assert.ok(stats.blocksUntilValidFinalize);
+      assert.ok(stats.hoursUntilValidFinalize);
+
```

Comment:
Small nit: I think a comment here would go really well with the comment you have below

Reply:
added in  4b10ba66e